demos/m5/README.md

```diff
@@ -2,9 +2,17 @@
 
 This directory contains the resources needed for running the release demo for Milestone 5.
 
-## Environments
+# Boostraping
 
-This demo can run in two different environments:
+After bootstrapping the cluster with App Studio, you can prepare the demo by running the [boostrap.sh](demos/m5/boostrap.sh) script.
+```
+$ ./demos/m5/boostrap.sh
```

Comment:
If you're in this directory reading this, you probably aren't going to do `./demos/m5/`

Reply:
You are right.